pages/common/tldr.md

```diff
@@ -7,9 +7,9 @@
 
 `tldr {{command}}`
 
-- Show the tar tldr page for Linux:
+- Show the scrot tldr page for Linux:
 
-`tldr -p {{linux}} {{tar}}`
+`tldr -p {{linux}} {{scrot}}`
 
 - Get help for a Git subcommand:
```

Comment:
```suggestion
- Get help for a `git` subcommand:
```

Reply:
@bl-ue no, we made a mass change to change `git` to `Git`.

Reply:
I remember the git → Git change, but here I'm emphasizing `git` the command rather than Git the software term.  Do you still think we should revert this change?